docs/en/Configuring-Alluxio-with-Azure-Blob-Store.md

```diff
@@ -80,7 +80,7 @@ $ ./bin/alluxio runTests
 After this succeeds, you can visit your container `AZURE_CONTAINER` to verify the files and directories created by Alluxio exist. For this test, you should see files named like:
 
 ```
-AZURE_DIRECTORY/default_tests_files/BASIC_CACHE_PROMOTE_CACHE_THROUGH
+AZURE_CONTAINER/AZURE_DIRECTORY/default_tests_files/BASIC_CACHE_PROMOTE_CACHE_THROUGH
```

Comment:
hout about replacing `AZURE_CONTAINER` with `<AZURE_CONTAINER>` 
and `AZURE_DIRECTORY` with `<AZURE_DIRECTORY>` through out this doc?
to emphasize that these are some "macros" users need to replace.

Reply:
Done.